configuration/pih/htmlforms/ncd-adult-initial.xml

```diff
@@ -1259,8 +1259,7 @@
                                 CHW: &amp;nbsp&amp;nbsp
                             </label>
                             <span>
-                                <obs id="chw-name" conceptId="CIEL:164141" style="person"
-                                     role="Application Role: communityHealthWorker"/>
+                                <obs id="chw-name" conceptId="CIEL:164141" style="provider"/>
```

Comment:
This will change this to a drop-down of all providers in the system, stored as an Obs using valueText.

To limit the list to only providers with a certain provider role, you first need to set up your provider roles and allocated them among providers, and then you can add the ```providerRoles="role-uuid"``` to the tag.

Reply:
Thanks @mseaton .  Looks good.